tmt/base.py

```diff
@@ -230,6 +230,29 @@ def _lint_summary(self):
             verdict(None, "summary should not exceed 50 characters")
         return True
 
+    def has_link(self, link_object):
+        """ Whether object contains specified link """
+        if isinstance(link_object, Link):
+            relation = list(link_object.keys())[0]
+            target = link_object[relation]
+        else:
+            # User text input
+            parts = link_object.split(':', maxsplit=1)
+            if len(parts) == 1:
+                relation, target = "relates", parts[0]
+            else:
+                relation, target = parts
+        for candidate in self.link:
+            candidate_relation = list(candidate.keys())[0]
```

Comment:
One rather corner case: If the link `note` is specified as the first in the link dictionary the search does not work. We should filter out notes first.

Reply:
Good point, I'll add test capturing this.